src/obu.rs

```diff
@@ -549,6 +541,47 @@ fn parse_seq_hdr(c: &mut Rav1dContext, gb: &mut GetBits) -> Rav1dResult<Rav1dSeq
     })
 }
 
+pub(crate) unsafe fn rav1d_parse_sequence_header(
+    mut data: &[u8],
+) -> Rav1dResult<DRav1d<Rav1dSequenceHeader, Dav1dSequenceHeader>> {
+    while !data.is_empty() {
```

Comment:
```suggestion
    let res = Err(ENOENT);

    while !data.is_empty() {
```

Reply:
I made `res` mutable so that it can be modified in the loop